PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -65,8 +88,9 @@ can be found in
 
     dspace/config/modules/swordv2-server.cfg
     
-By default it will crosswalk a broad selection of Dublin Core fields, and you can customise that here.  If you wish to also support RIOXX you may need to
-extend your EntryIngester implementation.
+By default it will crosswalk a broad selection of Dublin Core fields, and you can customise that here. [More info](https://wiki.duraspace.org/display/DSDOC5x/Metadata+and+Bitstream+Format+Registries)
+
+If you wish to also support RIOXX you may need to extend your EntryIngester implementation.
```

Comment:
Are you sure about that?

Why "...may..."? Are there circumstances where that isn't necessary. Any links to more guidance?

Reply:
it was already there, and I have no idea what changes are required for that, need to investigate it